BedrockCommand.cpp

```diff
@@ -184,6 +192,9 @@ void BedrockCommand::finalizeTimingInfo() {
     uint64_t unaccountedTime = totalTime - (prePeekTotal + peekTotal + processTotal + postProcessTotal + commitWorkerTotal + commitSyncTotal +
                                             escalationTimeUS + queueWorkerTotal + queueBlockingTotal + queueSyncTotal + queuePageLockTotal);
 
+    uint64_t blockingCommitThreadTime = blockingPrePeekTotal + blockingPeekTotal + blockingProcessTotal + blockingPostProcessTotal + blockingCommitWorkerTotal;
+    uint64_t exclusiveTransactionLockTime = blockingPeekTotal + blockingProcessTotal + blockingCommitWorkerTotal;
```

Comment:
Move this above `blockingCommitThreadTime`, and make `blockingCommitThreadTime` just be `exclusiveTransactionLockTime` + some other stuff.

Reply:
Done